src/pages/organize/[orgId]/campaigns/[campId]/surveys/[surveyId]/index.tsx

```diff
@@ -77,6 +78,10 @@ const SurveyPage: PageWithLayout<SurveyPageProps> = ({
           </Grid>
           <Grid item md={4}>
             <SurveyURLCard isOpen={isOpen} orgId={orgId} surveyId={surveyId} />
+            <SurveyUnlinkedCard
```

Comment:
This card (or it's contents) should only be visible when there are unlinked submissions. If there are zero, the card should not be vissible.

Reply:
in SurveyUnlinkedCard.tsx, I wrote unlinkedSubmitter > 0 && <Card/> so it wouldn't be visible when the count is 0.

Should I bring that count out of the SurveyUnlinkedCard component and use it to in index.tsx instead?